packages/components/table/src/table-column/index.ts

```diff
@@ -67,14 +67,16 @@ export default defineComponent({
 
       const type = props.type || 'default'
       const sortable = props.sortable === '' ? true : props.sortable
+      const showOverflowTooltip =
+        props.showOverflowTooltip || parent.props.showOverflowTooltip
```

Comment:
If we want to disable the overflow tooltip for single column, it won't work due to the `||` operator. For example,

```
<el-table show-overflow-tooltip>
  <el-table-column :show-overflow-tooltip="false" />
</el-table>
```

Reply:
Yes, this is indeed a problem.